src/rsz/src/SizeDownMove.cc

```diff
@@ -68,19 +79,11 @@ bool SizeDownMove::doMove(const Path* drvr_path,
       Vertex* fanout_vertex = edge->to(graph_);
       const Slack fanout_slack
           = sta_->vertexSlack(fanout_vertex, resizer_->max_);
-      Pin* load_pin = load_vertex->pin();
-      Instance* load_inst = network_->instance(load_pin);
-      debugPrint(logger_,
-                 RSZ,
-                 "size_down",
-                 3,
-                 " unsorted {} slack: {} drvr slack: {}",
-                 network_->pathName(fanout_vertex->pin()),
-                 delayAsString(fanout_slack, sta_, 3),
-                 delayAsString(drvr_slack, sta_, 3))
-          // If we already have a move on the load, don't try to size down
-          if (!hasMoves(load_inst))
-      {
+      Pin* fanout_pin = fanout_vertex->pin();
+      Instance* fanout_inst = network_->instance(fanout_pin);
+      // If we already have a move on the fanout gate, don't try to size down
+      // again
+      if (!hasMoves(fanout_inst)) {
```

Comment:
I know we have avoided combining moves on instances due to journaling limitations. These limitations are now gone. Do you think we should continue these checks?